index.js

```diff
@@ -29,6 +30,18 @@ const getEnv = ({env: envOption, extendEnv, preferLocal, localDir, execPath}) =>
 	return env;
 };
 
+const getFilePath = file => {
+	if (file instanceof URL) {
+		return fileURLToPath(file);
+	}
+
+	if (typeof file !== 'string') {
+		throw new TypeError('First argument must be a string or a file URL.');
+	}
```

Comment:
Catches potential mistake: `execa(['command', ...args])` instead of `execa('command', [...args])`.